api/tests/integration/infrastructure/repositories/thematic-repository_test.js

```diff
@@ -0,0 +1,25 @@
+const { expect, mockLearningContent, domainBuilder } = require('../../../test-helper');
+const Thematic = require('../../../../lib/domain/models/Thematic');
+const thematicRepository = require('../../../../lib/infrastructure/repositories/thematic-repository');
+
+describe('Integration | Repository | thematic-repository', function () {
+  describe('#list', function () {
+    it('should return the thematics', async function () {
+      // given
+      const thematic = domainBuilder.buildThematic({ id: 'recThematic1' });
+
+      const learningContent = {
+        thematics: [thematic],
+      };
+
+      mockLearningContent(learningContent);
+
+      // when
+      const actualThematics = await thematicRepository.list();
+
+      // then
+      expect(actualThematics.length).to.equal(1);
+      expect(actualThematics[0]).to.be.instanceOf(Thematic);
```

Comment:
Pour info, côté api il existe une fonction `deepEqualArray` qui permet de vérifier la taille d'un tableau, les instances des objets contenus dans ce tableau et fait un deepEqual sur ces objets (`expect(actualThematics).to.deepEqualArray([thematic])`). Ca te permettrait de tester aussi ici ta fonction `_toDomain`.